encryption/BIKE/BIKE/param/level-1.yaml

```diff
@@ -0,0 +1,13 @@
+name: Level 1
+comment: '(r, w, t) = (12323, 142, 134)'
+security level:
+  quantum: 64
+  nist category: 1
+  classical: 128
+failure probability: -128
+failure probability comment: !<tag:yaml.org,2002:js/undefined> ''
+sizes:
+  sk: 281
+  pk: 1541
+  ct|sig: 1573
+number of operations: '1'
```

Comment:
Since only the CPA proof is given, I would suggest that we write `CPA` with number of operations `inf`. And we should explicitly state in a comment that BIKE is only meant to be used in an ephemeral setting, as a friendly reminder.
(And link to: [How should BIKE be used?](https://bikesuite.org/files/v4.0/BIKE_Spec.2020.05.03.1.pdf#subsubsection.4.1.4))

Reply:
Thank you for the comments. I got a little bit confused since they mentioned in the paper that they achieved CCA security but only in a very [conditional way](https://bikesuite.org/files/v4.0/BIKE_Spec.2020.05.03.1.pdf#page=16&zoom=100,148,566). But yes, you are completely right and I am about to fix that issue.